addons/test_event_full/tests/test_event_mail.py

```diff
@@ -6,7 +6,10 @@
 
 from odoo.addons.mail.tests.common import MockEmail
 from odoo.addons.sms.tests.common import MockSMS
-from odoo.addons.test_event_full.tests.common import TestWEventCommon
+from odoo.addons.test_event_full.tests.common import (
+    TestWEventCommon,
+    TestEventFullCommon,
+)
```

Comment:
Don't waste lines ^^
```suggestion
from odoo.addons.test_event_full.tests.common import TestWEventCommon, TestEventFullCommon
```

Reply:
Suggested by the formatter/linter :sweat_smile: Is it wrong? (so I remove it also from linter config)